schemas/create-status.yml

```diff
@@ -0,0 +1,22 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:        "Create status"
+description: |
+  Create a commit status on GitHub
+type:         object
+properties:
+  state:
+    type: string
+    enum: ['pending', 'success', 'error', 'failure']
+    description: github status associated with the build.
```

Comment:
Maybe a link to where github defines these as well? This is rendered as markdown on our docs site so we can use that link syntax.